### DISCUSSION TITLE
Docker container frigate-dev usage

### DISCUSSION THREAD

**Discussion by herostrat** [blakeblackshear/frigate#1217] [Ask A Question] [answered]
As a mostly embedded guy I struggle a bit with the dev environment and how to use it properly.

The release-0.9.0 branch documents the basic usage, which seems to work. I think I get the basics of docker-compose.

atm my workflow is:
 * start containers
 * start frigate in frigate-dev container
 * write code in repo
 * restart containers to see changes
 * start frigate in frigate-dev container

This seems kinda complicated. Is there a way that my code changes in my local repo are mirrored in the frigate-dev container? Is it already the case and I just use it wrongly?

Thanks for the help.

**Comment by blakeblackshear** [comment, marked as answer]
If you are using visual studio code, it will prompt you to reopen in a remote container. Then you just edit the code and start frigate with `sudo python3 -m frigate` directly in the terminal within visual studio code.

**Comment by herostrat** [comment]
Note: I only tried using the shell, not opening the remote container -.-
Now it works and is really very nice to develop with.

Once again, thanks for the help!
